acceptance/tests/facts/windows_os.rb

```diff
@@ -0,0 +1,21 @@
+test_name "Test facter os reports correct os.windows.system32" do
+
+  confine :to, :platform => /windows/
+
+  @initial_boot = nil
+
+  step "Install Windows remote desktop feature" do
+    on(agent, powershell("Add-WindowsFeature –Name RDS-RD-Server –IncludeAllSubFeature"))
+    agent.reboot
+  end
+  step "Check is host is up" do
+    fail_test 'not up' unless agent.up?
+  end
```

Comment:
I think we can just skip this, it fail will because it will not be able to connect if the host is not up